drivers/mpu9250/MPU9250.hpp

```diff
@@ -34,11 +34,13 @@
 #pragma once
 
 #include "ImuSensor.hpp"
+#include <px4_defines.h>
 
 namespace DriverFramework {
 
 // update frequency 250 Hz
 #define MPU9250_MEASURE_INTERVAL_US 4000
+#define GYRO_RAW_TO_RAD_S 	32768.0f * 2000.0f * M_PI_2_F / 180.0f;		// 16 bit register, max 2000 degrees/s, deg to rad conversion
```

Comment:
The define should not have a semicolon at the and an expression like this should always be in brackets.